### ISSUE TITLE
For one line embedding allow shortcut `end=start`

### ISSUE THREAD

**Issue by alexander-yevsyukov** [SpineEventEngine/embed-code#19]
Currently, in order to embed one line of code we specify the same value for `start` and `end` parameters of the command. It would be good to allow some kind of shortcut to avoid specifying the same value for the `end`.

How about `lines=1` or `size=1`?

**Comment by alexander-yevsyukov** [comment]
Or it could be solved by having only one parameter named say, `line`, which accepts the same which `start` does.

**Comment by Vladyslav-Kuksiuk** [comment]
@alexander-yevsyukov,

Do we need this feature in the Go implementation?

**Comment by alexander-yevsyukov** [comment]
Yes. Please assume that if there is no `start` and `end` parameters we want only one line. Please also document this convention appropriately.
